webpack.config.js

```diff
@@ -4,9 +4,9 @@ const SRC_DIR = path.join(__dirname, '/client');
 const DIST_DIR = path.join(__dirname, '/public');
 
 module.exports = {
-  entry: `${SRC_DIR}/index.jsx`,
+  entry: `${SRC_DIR}/app.jsx`,
   output: {
-    filename: 'bundle.js',
+    filename: 'bundle-proxy.js',
```

Comment:
Sorry, would you be able to consider renaming to 'proxy-bundle.js'?

- recommend-bundle.js
- product-bundle.js
- inventory-bundle.js
- ...